LICENSE

```diff
@@ -1,26 +0,0 @@
-Allen Institute Software License – This software license is the 2-clause BSD license 
```

Comment:
?

Reply:
This must be the license thing that I ignored :/